Dockerfile

```diff
@@ -16,8 +15,7 @@ RUN dnf -y remove *subscription-manager* && \
       https://rpm.manageiq.org/release/14-najdorf/el8/noarch/manageiq-release-14.0-1.el8.noarch.rpm && \
     dnf -y module enable ruby:2.7 && \
     dnf -y module enable nodejs:12 && \
-    if [ ${ARCH} != "s390x" ] ; then dnf -y module disable virt:rhel; fi && \
-    dnf config-manager --setopt=ubi-8-*.exclude=rpm* --save && \
```

Comment:
Do we want to pull rpm* packages from the UBI repos?  This has caused build failures in the past where there is a newer version available in the UBI repos that has a dependency that is not yet available in the upstream CentOS repos.  Have you tested this on PPC and x86_64 to verify that the upstream and downstream are now in sync?

Reply:
I left this line in and it worked fine, so we can undo this line.

Reply:
We may need a conditional here.  When building on RHEL (and for s390x where there is no CentOS repo) we probably don't want this exclude.